Authors: Shivkumar Vishnempet Shridhar†, Francesco Beghini†, Marcus Alexander, Adarsh Singh, Ilana L. Brito*, and Nicholas A. Christakis*
By: Christakis Group (Yale), and Brito Group (Cornell), United States of America

## Honduras Microbiome phenotype project

This github repo describes workflow and codes used in Honduras microbiome study:

### Abstract:

Despite a growing interest in the gut microbiome of non-industrialized regions of the world, data linking microbiome features from such settings to diverse phenotypes remains uncommon. Using metagenomic data from a community-based cohort of 1,871 people from isolated villages in the Mesoamerican highlands of Western Honduras, we report 11,265 associations spanning 855 gut microbial species and 123 phenotypes. We report 593 associations with mental health phenotypes alone, as well as 4,276 associations with diverse socioeconomic phenotypes. Distinctly, we found 988 associations with microbiome metabolic pathways. We also report 10,653 associations after including strain-level phylogenies from 767 microbial species. Including such strain-phylogenetic information changes the overall relationship between the gut microbiome and these phenotypes, and strain-level phylogenetic information enhances the observed relationship between the microbiome and phenotypes as a whole. Our findings suggest new roles that gut microbiome surveillance can play in understanding broad features of individual and public health

### Contents:

- Phenotype-health associations
- Phenotype-food&animal associations 
- Phenotype-socioeconomic associations
- Supplementary figures
- Supplementary data
- Miscellaneous

### Species abundance profiles

Relative abundances of species and pathways generated by Metaphlan4 and Humann3 respectively after pre-processing.

### Microbiome-phenotype association

We evaluated associations between gut microbiome and phenotypes using linear mixed model.

  Species abundance ~ age + sex + BMI + batch effect + bristol stool scale + DNA concentration + Sampling date + 1|village + phenotype
  
where the relative abundance of species was transformed using the centred additive log-ratio (CLR) transformation.

### Microbiome-pathway association

We evaluated associations between metabolic pathways and phenotypes using linear mixed model.

  Pathway abundance ~ age + sex + BMI + batch effect + bristol stool scale + DNA concentration + Sampling date + 1|village + phenotype

### Calculation of microbiome variance explained by phenotypes and pathways

The microbiome composition variance explained by phenotypes was calculated by permutational multivariate analysis of variance using distance matrices, implemented in the adonis function for R package vegan (v.2.6), using 1000 permutations and a Bray-Curtis distance matrix calculated using relative abundances of microbial species. Variance explained was also performed using relative abundances of MetaCyc microbial biochemical pathways separately.

### Strain phenotype analysis

For strain-level analysis, we used the ANPAN package. Using the leaf distance in the phylogenetic tree, a linear mixed model – namely Phylogenetic Generalized Linear Mixed Model (ANPAN package v 0.2.0) – was implemented to get associations between phenotypes and strains:
Phenotype ~ Age + Sex + BMI + Batch effect + Bristol stool scale + DNA concentration+ Sampling date + Species(abundance) + 1|village + 1|leaf + ɛ

### Plotting scripts

Phenotype-health, food&animal, and socio-economic, and supplmentary figure folders contain reproducible code capable of generating the figures in the paper.

